assets/wizards/advertising/style.scss

```diff
@@ -1,4 +1,4 @@
-@import '~@wordpress/base-styles/colors';
+@use '~@wordpress/base-styles/colors' as wp-colors;
```

Comment:
I think it's safe to remove this, since none of the `wp-colors` variables are used in this file.

Reply:
Done in fa8dc6699d7f3726ec14ef3057596338ce7d230d